content/python/concepts/random-module/terms/uniform/uniform.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'uniform()'
+Description: 'Returns a pseudo-random floating-point number between two given numbers, inclusive.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Random'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.uniform()` method takes two numbers as arguments and returns a pseudo-random floating-point number between them, inclusive of the numbers provided.
+
+## Syntax
+
+```pseudo
+random.uniform(<low value>,<high value>)
```

Comment:
```suggestion
random.uniform(low_value, high_value)
```

I recommend this change because even though this is a `pseudo` code snippet, it should still be technically valid Python code from a syntax standpoint. 

The `< >` characters along with the spaces in `low value` and `high value` would cause `SyntaxError`s whereas the suggested changes could conceivably work if copy/pasted (with the arguments were defined ahead on the reader's end).